CONTRIBUTING.md

```diff
@@ -123,21 +123,13 @@ pnpm run lint
 
 For VS Code users, you can install the [Biome VS Code extension](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) to see lints while typing.
 
-# Versioning
-
-We use [changesets](https://github.com/changesets/changesets) to manage version. Currently, all rslib packages will use a fixed unified version.
-
-The release notes are automatically generated by [GitHub releases](https://github.com/web-infra-dev/rslib/releases).
-
 ## Releasing
 
-Repository maintainers can publish a new version of all packages to npm.
-
-Here are the steps to publish (we generally use CI for releases and avoid publishing npm packages locally):
-
-<!-- 1. [Create release pull request](https://github.com/web-infra-dev/rslib/actions/workflows/release-pull-request.yml). -->
+Repository maintainers can publish a new version of changed packages to npm.
 
-1. Run `pnpm generate-release-pr` to create a release pull request.
-2. [Run the release action on release branch](https://github.com/web-infra-dev/rslib/actions/workflows/release.yml).
-3. Merge the release pull request.
-4. [Create Git tag on main branch and generate the release notes](https://github.com/web-infra-dev/rslib/releases).
+1. Checkout a new release branch, for example `release_v1.2.0`
```

Comment:
I wrote a script in Rslib that contains step 1 and 2

Reply:
nice

Reply:
done